variables.tf

```diff
@@ -255,6 +255,15 @@ DESCRIPTION
   }
 }
 
+variable "outbound_rules" {
+  type = map(object({
+    resource_id = string
+    type        = string
+  }))
+  default     = null
+  description = "Map of outbound rules for the managed network"
```

Comment:
Might be worth to add a description to the fields of the object that it is created here.

Reply:
added